src/plugins/appeals/unbanThread.ts

```diff
@@ -0,0 +1,35 @@
+import {useEvent} from "../../hooks";
+import {Interaction, Message} from "discord.js";
+
+useEvent("interactionCreate", async (interaction: Interaction) => {
+    if (!interaction.isButton()) {
+        return;
+    }
+    const {customId} = interaction;
+    const [interactionType, action, userTag, userId] = customId.split("-");
+    const {message} = interaction;
+    if (interactionType !== "appeal") {
+        return;
+    }
+    if (action !== "thread") {
+        return;
+    }
+    try {
+        await interaction.message.startThread ({
```

Comment:
It would be better to use the already defined `message` variable than to access it from the `interaction` object again.

Reply:
Change made.